addons/website_event_track/static/src/js/website_event_track_proposal.js

```diff
@@ -0,0 +1,185 @@
+odoo.define('website_event_track.website_event_track_proposal', function (require) {
```

Comment:
Reminder: to update (with file name) to match name of widget (if widget is called eventDoStuff, here this will be event_do_stuff and file event_do_stuff.js).